collector/src/test/java/com/google/cloud/trace/zipkin/PlainTextNettyChannelProvider.java

```diff
@@ -0,0 +1,37 @@
+package com.google.cloud.trace.zipkin;
+
+import com.google.auto.service.AutoService;
+import io.grpc.ManagedChannelProvider;
+import io.grpc.netty.NettyChannelBuilder;
+
+/**
+ * Allows non-ssl grpc connections for {@link NettyChannelBuilder} instances.
+ */
+@AutoService(ManagedChannelProvider.class)
+public class PlainTextNettyChannelProvider extends ManagedChannelProvider
+{
+  @Override
+  public boolean isAvailable()
+  {
+    return true;
+  }
+
+  @Override
+  public int priority()
+  {
+    //put it in front of all other providers: NettyChannelProvider/OkHttpChannelProvider
```

Comment:
That should work. You could also have a simple test implementation of TraceSink that calls the StackdriverMockServer directly. Once we've updated the SDK, we can change it to use the real GrpcTraceSink.